src/main/java/it/pagopa/pn/deliverypush/service/impl/WebhookEventsServiceImpl.java

```diff
@@ -161,6 +161,7 @@ private Mono<Void> saveEventWithAtomicIncrement(StreamEntity streamEntity, Strin
 
                 return eventEntityDao.save(webhookUtils.buildEventEntity(atomicCounterUpdated, streamEntity,
                         newStatus, timelineElementInternal, notificationInt))
+                        .onErrorResume(ex -> Mono.empty())
```

Comment:
onerror, log e throw.
in generale lasciar sempre salire le exceptions

Reply:
modifica effettuata per la gestione dell'errore